### PR TITLE
Copy SSL configuration file for apache

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-bug-id: Tendrl/api#264
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , Please review

File: tendrl-api.spec [hunk 1]
```diff
@@ -75,7 +75,7 @@ install -Dm 0644 lib/tendrl/errors/*.rb $RPM_BUILD_ROOT%{_datadir}/%{name}/lib/t
 install -Dm 0644 tendrl-api.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-api.service
 install -Dm 0644 config/etcd.sample.yml $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/etcd.yml
 install -Dm 0644 README.adoc Rakefile $RPM_BUILD_ROOT%{_datadir}/doc/tendrl
-install -Dm 0644 config/apache.vhost.sample $RPM_BUILD_ROOT%{_sysconfdir}/httpd/conf.d/tendrl.conf
+install -Dm 0644 config/apache.vhost.sample $RPM_BUILD_ROOT%{_sysconfdir}/httpd/conf.d/tendrl-ssl.conf.sample
```
Comment (shtripat) [inline tendrl-api.spec]: the deployed file is `tendrl-ssl.conf.sample`  ?

Comment (nthomas-redhat) [reply tendrl-api.spec]: @shtripat , this is inline with https://github.com/Tendrl/api/pull/264

**Comment by nthomas-redhat** [comment]
@brainfunked , Please take a look before the merge

**Pull request merged**